src/pages/en/toberfests.mdx

```diff
@@ -0,0 +1,96 @@
+
+![Screenshot showing the Branch Selection interface of Visual Studio Code](/assets/img/writing/toberfest/toberfest-5.png)
+### Pull Requests
+The last thing you need to know about Git and GitHub is the concept of Pull Requests. Pull requests are GitHub's system for sending changes between branches, or, more importantly, from a fork of a repository back to the original one. To create a Pull Request, visit your fork on GitHub, click current branch in the top left, and select the branch with your changes to switch to it.
+
+![Screenshot highlighting the branch switching interface on github.com](/assets/img/writing/toberfest/toberfest-6.png)
+
+Then click "Contribute" and then click "Create Pull Request". More information about what you need to include in your Pull Requests can be found further down the page.
+
+And that is the basics of how to use git and GitHub. If you're interested in learning more about how git works, there is an excellent visual tutorial available at [https://learngitbranching.js.org](https://learngitbranching.js.org). If you get stuck, don't be afraid to ask for help on [Discord](https://discord.quiltmc.org/toolchain).
+
+## Finding Something to Contribute
+The following Quilt projects are accepting **code** Pull Requests for Hacktoberfest and Modtoberfest:
+- [Chasm](https://github.com/quiltmc/chasm)
+- [Developer Wiki](https://github.com/quiltmc/developer-wiki)
+- [Enigma](https://github.com/quiltmc/enigma)
+- [Mappings Hasher](https://github.com/quiltmc/mappings-hasher)
+- [Quilt Loom](https://github.com/quiltmc/quilt-loom)
+- [quiltmc.org](https://github.com/quiltmc/quiltmc.org)
+- [Quilted Fabric API](https://github.com/quiltmc/quilted-fabric-api)
+- [Quiltflower](https://github.com/quiltmc/quiltflower)
+- [Quilt Mappings](https://github.com/quiltmc/quilt-mappings)
+- [Quilt Standard Libraries](https://github.com/quiltmc/quilt-standard-libraries)
+
+In addition, the following Quilt projects are accepting **low or non-code** Pull Requests for Hacktoberfest:
+- Developer Wiki (documentation)
+- quiltmc.org (Translation)
+
+After you decide on a repository to contribute to, a great place to start is by looking at the Issues tab for something you think you could help with, unless you already have something in mind. If you find something that you think you can do, please leave a comment stating your intent to work on it, and please clarify any details you're unsure of before starting work. Likewise, if you're attempting something that isn't being tracked in an issue, **please** confirm that it is something that would be beneficial to the project before you start, either by making an issue on GitHub or chatting to the developers on [Discord](https://discord.quiltmc.org/toolchain); we'd hate for your work to go to waste! We will not accept contributions that are obviously low-effort or unhelpful, and PRs will be accepted at the sole discretion of the team responsible for the repository.
```

Comment:
Should use the `discordToolchain` variable from `@data/Site`
```diff
- [Discord](https://discord.quiltmc.org/toolchain)
+ <a href={site.discordToolchain}>Discord</a>
```
Also I feel like we should clarify that we have two Discord servers? Currently that is not very clear

Reply:
I'm only linking to Toolchain because it's the one focused on development, so I think it's OK.